<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Võ Bích Ngọc K23111406</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRefresh">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Refresh Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Add New Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Delete Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSort">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Sort By Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerate">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Generate Chart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddUSD">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>USD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGbyG">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Group </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
